KaraIOke/ViewModels/PlaylistListViewModel.cs

```diff
@@ -0,0 +1,55 @@
+using System.ComponentModel;
+using System.Windows.Input;
+using System.Collections.ObjectModel;
+using KaraIOke.Services.AppEnvironment;
+using KaraIOke.Services.Navigation;
+
+namespace KaraIOke.ViewModels;
+
+public class PlaylistListViewModel : INotifyPropertyChanged
```

Comment:
`PlaylistList` feels wrong, maybe we can rename `PlayListList` to `Playlists` and then a single `Playlist` would be `PlaylistDetail`?